Libraries/TestHelpers.psm1

```diff
@@ -760,8 +760,19 @@ Function Get-LISAv2Tools($XMLSecretFile) {
 		if (! (Test-Path $CurrentDirectory/Tools/$_) ) {
 			Write-LogWarn "$_ file is not found in Tools folder."
 			if ($toolFileAccessLocation) {
-				$WebClient.DownloadFile("$toolFileAccessLocation/$_","$CurrentDirectory\Tools\$_")
-				Write-LogInfo "File $_ successfully downloaded in Tools folder: $CurrentDirectory\Tools."
+				$downloadFileError = $False
+				try {
+					$WebClient.DownloadFile("$toolFileAccessLocation/$_","$CurrentDirectory\Tools\$_")
+				}
+				catch {
+					$downloadFileError = $True
+				}
+				if ($downloadFileError) {
+					Write-LogWarn "Failed to download '$_', please make sure it's available from '$toolFileAccessLocation'"
```

Comment:
Should it stop the run-time if failed?

Reply:
https://github.com/LIS/LISAv2/issues/939

Reply:
I see, I am not aware of this communication. As it is a protected file, we should continue then.